Students/Naomi/ObjcTextInput/ObjcTextInput/Modules/TextInput/TIIUIViewController.m

```diff
@@ -84,7 +84,12 @@ - (void) prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 
 -(void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPath:(NSIndexPath *)indexPath  {
     self.indexPathRow = indexPath.row;
-    [self performSegueWithIdentifier:@"editNameSegue" sender:self];
```

Comment:
Waarom doe je dat? De segue bestaat niet als je hem in een andere context gebruikt? Het komt er eigenlijk op neer dat je de perform niet in code maar in de storyboard moet doen.